### DISCUSSION TITLE
en Daily.dev antes traducia

### DISCUSSION THREAD

**Discussion by mendozalz** [dailydotdev/daily#937] [Feature Requests]
en Daily.dev antes traducía las cards, tanto en firefox como en google chrome, en mi teléfono android si lo hace, pero en el navegador ya no! Que cambio ha pasado o como se puede solucionar este inconveniente?

PD: No falta el habido que me salga ES IMPERANTE APRENDER INGLES, y mi respuesta es que mientras lo sigo aprendiendo no quiero perderme nada en la actualidad.

**Comment by idoshamun** [comment]
Hi @3P4R4,

Please write your request in English. I don't speak Spanish and can't understand your comment 🙏
